### DISCUSSION TITLE
How to rename repo and package name?

### DISCUSSION THREAD

**Discussion by nowsprinting** [openupm/openupm#4823] [Q&A] [answered]
I planned to rename my own package and repo name.

What happens to the package already registered in openupm when I run this?
For existing users, I would like the package before the change to remain.

**Comment by favoyang** [comment]
If you change the repo name only, the GitHub handle the redirect, you can optional update the package meta file's repo field on OpenUPM.

If you change the package name, you can rename the package meta file, and update the name field. It will not affect the published version on our registry.

**Comment by favoyang** [comment, marked as answer]
Learn more at https://openupm.com/docs/modifying-upm-package.html#renaming-a-package

**Comment by nowsprinting** [comment]
Thanks!
